doc/source/data/dataset.rst

```diff
@@ -29,21 +29,23 @@ is already supported.
   https://docs.google.com/drawings/d/16AwJeBNR46_TsrkOmMbGaBK7u-OPsf_V8fHjU-d2PPQ/edit
 
 
+----------------------------------------------
 Data Loading and Preprocessing for ML Training
-==============================================
+----------------------------------------------
 
-Ray Datasets is designed to load and preprocess data for distributed :ref:`ML training pipelines <train-docs>`.
+Ray Datasets are designed to load and preprocess data for distributed :ref:`ML training pipelines <train-docs>`.
```

Comment:
Hmm I don't agree with this change, "Ray Datasets" here is referring to the library, which is singular. We typically use "Datasets" or "`Dataset`s` to refer to the abstraction, which is plural.

For example, Hugging Face Datasets refers the library ("Hugging Face Datasets" and "Datasets") as a singular noun: https://huggingface.co/docs/datasets/index

>🤗 Datasets is a library for easily accessing and sharing datasets for Audio, Computer Vision, and Natural Language Processing (NLP) tasks.

Reply:
This is a good point, especially given the huggingface example.

Reply:
Ah, yeah I did realize this was the intention but wasn't sure if this would make sense from an external user's perspective since we never explicitly differentiate the library and the abstraction in this page 🥲. For example, the first sentence in the page is:

> Ray Datasets are the standard way to load and exchange data in Ray libraries and applications.

And in general the existing inconsistencies between how Ray Datasets and Datasets are used interchangeably got me confused.

I can take another pass over it following the suggested pattern.